File: front/src/dto/riyousha/saveRiyoushaComradeCapsuleDto.ts
import FrameworkCapsuleDto from '../frameworkCapsuleDto';
import RiyoushaComradeDto from './riyoushaComradeDto';


export default interface SaveRiyoushaComradeCapsuleInterface {
}


/**
 * APIユーザ格納Dto
 */
export default class SaveRiyoushaComradeCapsuleDto extends FrameworkCapsuleDto
  implements SaveRiyoushaComradeCapsuleInterface {
  /**
   * 利用者API接続Dto
   */
  riyoushaComradeDto: RiyoushaComradeDto;

  constructor() {
    super();
    this.riyoushaComradeDto = new RiyoushaComradeDto;
  }
}

File: back/src/main/java/mitei/mitei/political/balancesheet/manage/kanrensha/dto/riyousha/SaveRiyoushaComradeCapsuleDto.java
package mitei.mitei.political.balancesheet.manage.kanrensha.dto.riyousha;

import java.io.Serializable;

import mitei.mitei.political.balancesheet.manage.kanrensha.dto.FrameworkCapsuleDto;

/**
 * APIユーザ格納Dto
 */
public class SaveRiyoushaComradeCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 利用者API接続Dto */
    private RiyoushaComradeDto riyoushaComradeDto = new RiyoushaComradeDto();

    /**
     * 利用者API接続Dtoを取得する
     *
     * @return 利用者API接続Dto
     */
    public RiyoushaComradeDto getRiyoushaComradeDto() {
        return riyoushaComradeDto;
    }

    /**
     * 利用者API接続Dtoを設定する
     *
     * @param riyoushaComradeDto 利用者API接続Dto
     */
    public void setRiyoushaComradeDto(final RiyoushaComradeDto riyoushaComradeDto) {
        this.riyoushaComradeDto = riyoushaComradeDto;
    }

}
